test/integration/short/client-each-row-tests.js

```diff
@@ -567,10 +567,15 @@ function insertTestData(client, table, length, callback) {
 }
 
 function insertSelectTest(table, rowLength, times, selectLimit, selectOptions, done) {
-  var client = newInstance({ queryOptions: {
-    consistency: types.consistencies.quorum,
-    fetchSize: rowLength
-  }});
+  var client = newInstance({
+    queryOptions: {
+      consistency: types.consistencies.quorum,
+      fetchSize: rowLength
+    },
+    socketOptions: {
+      readTimeout: 100000
```

Comment:
This is probably a good idea given how unreliable the CI environments we use can be at times.